src/vivarium_ciff_sam/components/x_factor.py

```diff
@@ -19,3 +22,47 @@ def get_randomness_stream(self, builder) -> RandomnessStream:
 
     def register_simulant_initializer(self, builder: Builder) -> None:
         pass
+
+
+class XFactorEffect(RiskEffect):
+
+    configuration_defaults = {
+        'effect_of_risk_on_target': {
+            'conditional_exposure': 0.0,
+            'measure': {
+                'relative_risk': None,
+                'mean': None,
+                'se': None,
+                'log_mean': None,
+                'log_se': None,
+                'tau_squared': None
+            }
+        }
+    }
+
+    def __init__(self, target: str):
+        super().__init__('risk_factor.x_factor', target)
+
+    ##########################
+    # Initialization methods #
+    ##########################
+
+    def get_configuration_defaults(self) -> Dict[str, Dict]:
+        return {
+            f'effect_of_{self.risk.name}_on_{self.target.name}': {
+                'conditional_exposure': XFactorEffect.configuration_defaults['effect_of_risk_on_target']['exposure'],
```

Comment:
Why is it just 'exposure' at the end of this line and not 'conditional_exposure'

Reply:
you're correct that's a typo